apps/atlas/src/app/search/details/components/properties/properties.component.html

```diff
@@ -1,15 +1,30 @@
-<models4insight-sortable-table-shell [config]="tableConfigProperties" [data]="dataForTable$ | async"
-  [itemsPerPage]="1000" [rowsSelectable]="false">
-  <ng-template let-rowData>
-    <td *ngIf="rowData">
-      <ng-container>
-        <div>
-          {{ rowData.name |translate | titlecase }}
-        </div>
-      </ng-container>
-    </td>
-    <td>
-      <models4insight-properties-item [data]="rowData.value"></models4insight-properties-item>
-    </td>
-  </ng-template>
-</models4insight-sortable-table-shell>
\ No newline at end of file
+<div class="collapsible-header" (click)="isCollapsed = !isCollapsed">
+  <h3 class="title is-3">Properties</h3>
+  <span class="arrow">
+    <svg width="16" height="16" viewBox="0 0 16 16" fill="currentColor">
+      <path *ngIf="isCollapsed" d="M1.646 4.646a.5.5 0 0 1 .708 0L8 10.293l5.646-5.647a.5.5 0 0 1 .708.708l-6 6a.5.5 0 0 1-.708 0l-6-6a.5.5 0 0 1 0-.708z"/>
+      <path *ngIf="!isCollapsed" d="M7.646 4.646a.5.5 0 0 1 .708 0l6 6a.5.5 0 0 1-.708.708L8 5.707l-5.646 5.647a.5.5 0 0 1-.708-.708l6-6z"/>
+    </svg>
+  </span>
+</div>
+<div [hidden]="isCollapsed">

```

Comment:
You can make this an `*ngIf` to avoid rendering the table at all when this section is collapsed. I suspect if you do that, you will also not need this weird timeout function anymore 😁

```suggestion
<div *ngIf="!isCollapsed">
```